packages/sdks/e2e/react-native-76-fabric/app/[...slug].tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Content,
+  _processContentResult,
+  fetchOneEntry,
+} from '@builder.io/sdk-react-native';
+import { getProps } from '@sdk/tests';
+import { useLocalSearchParams } from 'expo-router';
+import { useEffect, useState } from 'react';
+import { Text, View } from 'react-native';
+
+export default function HomeScreen() {
+  const [props, setProps] = useState<any>(undefined);
```

Comment:
Question: Can we use `BuilderContent` type here?
from import type { BuilderContent } from '@builder.io/sdk-react-native';

Reply:
`prop` is more than just `content`. The `props` value stored here is the output from `getProps()`, a test mock generator here:

https://github.com/BuilderIO/builder/blob/ecaf113e51354ff992cd3f8999d071992afbebb5/packages/sdks-tests/src/specs/index.ts#L367-L380

Notice `...extraProps`, and `as any` at the end. I tried to provide strict types but due to how flexible the output of `getProps` is, I ran into some issues (which I cannot exactly recall anymore).